dora/job/common/src/main/java/alluxio/scheduler/job/Task.java

```diff
@@ -170,7 +170,7 @@ public boolean equals(Object obj) {
       return false;
     }
     Task<?> other = (Task<?>) obj;
-    return getPriority() == other.getPriority()
+    return getTaskId() == other.getTaskId() && getPriority() == other.getPriority()
```

Comment:
@lucyge2022 @jja725 please review

Reply:
u can remove the priority, just use task id and job is enough

Reply:
ditto